### PR TITLE
initial connection to MongoDB, with a post request; also updated some…

### PR DESCRIPTION
**Pull request by zulnam**
In this PR:
* created initial connection to MongoDB with a basic post request;
* updated some packages.

### CODE REVIEW

File: packages/backend/mongo/sessionSchema.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+const mongoose = require('mongoose');
+
+const sessionSchema = new mongoose.Schema({
+  teamName: String,
+  roomName: String
+});
+
+Session = mongoose.models['tests'] || mongoose.model('tests', sessionSchema);
```
Comment (dominiksipowicz) [inline packages/backend/mongo/sessionSchema.js]: what are tests model?

File: pages/api/roomSession.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const apiNewSession = async (req, res) => {
+  try {
+    await newSession(req.body.teamName, req.body.roomName);
+    res.status(200);
+  } catch (error) {
+    console.log(error.message);
+  }
+};
+
+export default apiNewSession;
```
Comment (dominiksipowicz) [inline pages/api/roomSession.js]: Why do you call it "session"? and the file is called roomSession and you return apiNewSession also

**Review by dominiksipowicz — APPROVED** [review]

File: packages/backend/mongo/sessionSchema.js (hunk 1 above)
Comment (zulnam) [reply packages/backend/mongo/sessionSchema.js]: It's the temporary name of the model for the room declaration schema. Once I'll have more data to push to mongoDB i'll rename it.

File: pages/api/roomSession.js (hunk 2 above)
Comment (zulnam) [reply pages/api/roomSession.js]: Good point.

**Comment by zulnam** [comment]
MongoDB will not be part of this repo (frontend), but a data middleware.

**Pull request closed without merging**